### PR TITLE
feat: Create Fast API handler

### PR DESCRIPTION
**Pull request by StiVit**

### CODE REVIEW

File: test.db
Comment (grumpycatyo-collab) [inline test.db:1]: Make sure the `test.db` is also in the .gitignore files. You will need it locally only.

Comment (StiVit) [reply test.db:1]: 🫡

File: run.py
Comment (grumpycatyo-collab) [inline run.py:1]: I guess this uvicorn part should be in the `main.py`. From my point of view `run.py` is not the correct naming. Also you should move the `main.py` outside. I mean, it shouldn't be inside the `app` folder.

Comment (grumpycatyo-collab) [reply run.py:1]: So basically your `main.py` should be looking like this:
```
import uvicorn
from fastapi import FastAPI
from app.database import engine, Base
from app.routers import items

Base.metadata.create_all(bind=engine)
# FastAPI app instance
app = FastAPI()


# Include routers
app.include_router(items.router, prefix="/api", tags=["items"])

if __name__ == "__main__":
    uvicorn.run("app.main:app", host = "127.0.0.1", port=8000, reload=True)
```

**Review by grumpycatyo-collab — CHANGES_REQUESTED** [review]

File: README.md
Comment (grumpycatyo-collab) [inline README.md:1]: Update the documentation with your updates ( how the API works )

File: app/models.py
Comment (grumpycatyo-collab) [inline app/models.py:1]: The implementation with the `Item` model is good, however we need something more advanced, that's why I'd recommend to change the `Item` model towards the `User` model, and add some fields. Don't forget make changes in `schemas.py` and also other files where your model is mentioned

**Review by grumpycatyo-collab — APPROVED** [review]

**Pull request merged**